engine/voxel_engine_gd.cpp

```diff
@@ -177,6 +177,19 @@ Dictionary VoxelEngine::get_stats() const {
 	return to_dict(zylann::voxel::VoxelEngine::get_singleton().get_stats());
 }
 
+int VoxelEngine::get_thread_count() const {
+	return zylann::voxel::VoxelEngine::get_singleton().get_thread_count();
+}
+
+void VoxelEngine::set_thread_count(int count) {
+	ERR_FAIL_COND_MSG(count < 1,
+			vformat("The thread count must be a number from 1 to %d", ThreadedTaskRunner::MAX_THREADS));
+	// if `thread_count` is bigger than MAX_THREADS we allow the function to continue,
+	// as the rest of the code is build to handle this
```

Comment:
Except it is silently handled, and it is preferable to not assume that, you should put an error check.